.azure-pipelines/templates/run-tests.yml

```diff
@@ -8,13 +8,13 @@ parameters:
 steps:
 - ${{ if eq(parameters.test, 'true') }}:
   - script: ddev test --cov --junit ${{ parameters.check }}
-    displayName: 'Run Unit/Integration tests'
+    displayName: 'Run unit integration tests'
```

Comment:
What's `unit integration`?

Reply:
Because it does look nice here with `/`:

![image](https://user-images.githubusercontent.com/49917914/67714599-82846180-f9c8-11e9-94e0-c7ca1b5dae01.png)

Reply:
I think is clearer with the '/', maybe put a space `unit / integration`